packages/shared/src/components/feeds/FeedSettings/FeedSettingsEdit.tsx

```diff
@@ -113,47 +112,45 @@ export const FeedSettingsEdit = ({
   const router = useRouter();
   const feedSettingsEditContext = useFeedSettingsEdit({ feedSlugOrId });
   const { feed, onBackToFeed } = feedSettingsEditContext;
-  const { showPlusSubscription } = usePlusSubscription();
 
   const tabs = useMemo(() => {
     const base: TabOptions[] = [
       {
         title: feedSettingsMenuTitle.general,
         options: { icon: <EditIcon size={IconSize.Small} /> },
       },
+      {
+        title: feedSettingsMenuTitle.tags,
+        options: { icon: <HashtagIcon size={IconSize.Small} /> },
+      },
+      {
+        title: feedSettingsMenuTitle.sources,
+        options: { icon: <AddUserIcon size={IconSize.Small} /> },
+      },
+      {
+        title: feedSettingsMenuTitle.preferences,
+        options: { icon: <AppIcon size={IconSize.Small} /> },
+      },
+      {
+        title: feedSettingsMenuTitle.ai,
+        options: { icon: <MagicIcon size={IconSize.Small} /> },
+      },
```

Comment:
Don't we need to check for `feed?.type === FeedType.Main`?

Reply:
The condition was `OR`. With the value of `showPlusSubscription` being true now, that means, regardless whether it is main, this will proceed which is the case for users enrolled.

Reply:
Right, mb!